README.md

```diff
@@ -77,10 +80,13 @@ Called whenever new messages are received. `messages` is an array containing the
 Called whenever the support agent starts or stops typing
 
 #### onError(error: ?ApiError) => [QuiqChatClient](#quiqchatclient)
-Called whenever there is an error from the API
+Called whenever there is a non-retryable error or an error that has exceeded the maximum number of retries from the API.
+
+#### onRetryableError(error: ?ApiError) => [QuiqChatClient](#quiqchatclient)
```

Comment:
This is my oversight from before, but should we add the `ApiError` type to the documentation?